jenkins/infra/jenkins/build-ci-image-all.groovy

```diff
@@ -43,49 +43,63 @@ pipeline {
 
 		stage('build ci images') {
 			parallel {
-				stage('cpp') {
+				stage('cpp - ubuntu:22.04') {
```

Comment:
not sure if we need `ubuntu:22.04` in description vs something like more general like `default` or nothing, as latter would make updating default Ubuntu OS fewer changes

Reply:
yea, I thought the same also.